.ci/google-java-format.sh

```diff
@@ -15,7 +15,7 @@ INPUT_PATHS=($(find src/it/resources/com/google/checkstyle/test/ -name "Input*.j
     | grep -v "rule53camelcase" \
     | grep -v "rule522classnames" \
     | grep -v "rule41" \
-    | grep -v "rule42" \
+    | grep -v "rule42/ClassWithChainedMethods.java" \
```

Comment:
diff for ClassWithChainedMethods.java:

```diff
C:\OpenSource\checkstyle git:[15449-rule42]
git diff src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/ClassWithChainedMethods.java
diff --git a/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/ClassWithChainedMethods.java b/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/ClassWithChainedMethods.java
index 67fbdfbe8..202b2a53c 100644
--- a/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/ClassWithChainedMethods.java
+++ b/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/ClassWithChainedMethods.java
@@ -9,15 +9,15 @@ public class ClassWithChainedMethods {
   public static void main(String[] args) {
     new ClassWithChainedMethods()
         .getInstance("string_one")
-    .doNothing("string_one".trim(), "string_two");
+        .doNothing("string_one".trim(), "string_two");:wa
     // violation above ''method call' child has incorrect indentation level 4, expected .* 8.'
 
     int length =
-    new ClassWithChainedMethods("param1", "param2").getInstance().doNothing("nothing").length();
+        new ClassWithChainedMethods("param1", "param2").getInstance().doNothing("nothing").length();
     // violation above ''new' has incorrect indentation level 4, expected .* 8.'
 
     int length2 =
-    new ClassWithChainedMethods("param1", "param2").getInstance().doNothing("nothing").length();
+        new ClassWithChainedMethods("param1", "param2").getInstance().doNothing("nothing").length();
     // violation above ''new' has incorrect indentation level 4, expected .* 8.'
   }

```

after formatted code test started failing so I had to grep it

Reply:
Thanks  a lot. Good decision.